### PR TITLE
[GridView] Refactor "Working with Data" & "Adding New Entries" articles

### PR DESCRIPTION
**Pull request by dilyantraykov**

### CODE REVIEW

File: controls/radgridview/getting-started/working-with-data.md [hunk 1]
```diff
@@ -10,11 +10,17 @@ position: 2
...
+* [Loading Data](#loading-data)
+
+* [Displaying Data](#loading-data)
+
+* [Managing Data](#loading-data)
 
 ## Loading Data
 
-To load the data means to fetch it to the __RadGridView__, so it can display it. The loading part is most commonly associated with consuming a web service or reading a local data source. To learn more about the different ways of loading data take a look at the topics about loading data:
+__RadGridView__ provides you with a variety of different ways to populate it with data, so it can display it. Here's a list of possible ways you can populate your RadGridView:
```
Comment (PetyaSotirova) [inline controls/radgridview/getting-started/working-with-data.md]: Test comment

**Comment by PetyaSotirova** [comment]
General comment

File: controls/radgridview/managing-data/add-new-entries.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ The next step in implementing the adding functionality is to attach event handle
 
 ## Modifying New Entries
 
-The __AddingNewDataItem__ event is raised before the new row is added to the RadGridView. A typical use case is when you need to set initial values for the initialized object. This can be done by passing an object to the __GridViewAddingNewEventArgs__'s **NewObject** property.
```
Comment (dilyantraykov) [inline controls/radgridview/managing-data/add-new-entries.md]: @pramodprogress, in this case, the event is raised by RadGridView and not by the developer.
The same applies for line 113.

File: controls/radgridview/managing-data/add-new-entries.md [hunk 3]
```diff
@@ -34,13 +34,13 @@ There are two ways to insert a new row with RadGridView - by pressing the __Inse
...
+When a user adds a new item, an empty row is created in which the user can input data.
 
 #### __Figure 1: The newly created row__
 
 ![The newly created row](images/RadGridView_AddingNewItems_1.png)
 
->If the __IsReadOnly__ property of the RadGridView is set to __True__ or the __CanUserInsertRows__ property is set to __False__, no row will be added.
+>If the __IsReadOnly__ property of RadGridView is set to __True__ or the __CanUserInsertRows__ property is set to __False__, no row is added.
 
 The next step in implementing the adding functionality is to attach event handlers to the __AddingNewDataItem__ and the __RowEditEnded__ events.
```
Comment (dilyantraykov) [inline controls/radgridview/managing-data/add-new-entries.md]: I do not believe a separate section is necessary, but I will move this paragraph in the "Modifying New Entries" section.

**Pull request merged**